code/datums/mapgen/Cavegens/IcemoonCaves.dm

```diff
@@ -1,6 +1,9 @@
 /datum/map_generator/cave_generator/icemoon
 	weighted_open_turf_types = list(/turf/open/misc/asteroid/snow/icemoon = 19, /turf/open/misc/ice/icemoon = 1)
-	weighted_closed_turf_types = list(/turf/closed/mineral/random/snow = 1)
+	weighted_closed_turf_types = list(
+		/turf/closed/mineral/snowmountain/icemoon = 100,
+		/turf/closed/mineral/gibtonite/ice = 4,
+	)
```

Comment:
I dont want to sound stupid... but why do you changed it? Clearly that's what prevents IceBox to spawn ores isn't it?